source/agora/cli/client/SendTxProcess.d

```diff
@@ -124,11 +124,12 @@ public void printSendTxHelp (ref string[] outputs)
 
     Params:
         args = client command line arguments
+        outputs = Where to output any send output (1 line will be one entry)
```

Comment:
Isn't it to be `any sent output`?

Reply:
Reading again, that doc is terrible. Will fix.

Reply:
Done.